src/services/consumer-groups.ts

```diff
@@ -31,8 +29,8 @@ const getConsumerGroups = async (
   );
   const response: AxiosResponse<ConsumerGroupList> =
     await api.getConsumerGroups(
-      offset,
-      limit,
+      undefined,
```

Comment:
Not sure I'm following, doesn't the API support pagination at all?

Reply:
yes, api supports pagination. API doesn't return correct response with `offset`. Same issue I fixed in Topics API which I discussed with Wojciech and he told me not to use offset. With offset API doesn't return `total` in response, it return count which is not correct.
It's working fine.

Reply:
That is be cause offset limit and count are deprecated fields for all kafka api

https://github.com/bf2fc6cc711aee1a0c2a/kafka-admin-api/blob/main/kafka-admin/.openapi/kafka-admin-rest.yaml#L1612

What we should do is use page and size instead.

Reply:
If we see any field missing lets create issue so we can fix it on backend

Reply:
@wtrocki Thanks for adding details about offset. yes, we are already passing page and size in API call and it's working fine. 
There is no issue for now.

Reply:
This is about return data.
Parameters:
https://github.com/bf2fc6cc711aee1a0c2a/kafka-admin-api/blob/main/kafka-admin/.openapi/kafka-admin-rest.yaml#L256-L271

 Returned fields to use in pagination:
 https://github.com/bf2fc6cc711aee1a0c2a/kafka-admin-api/blob/main/kafka-admin/.openapi/kafka-admin-rest.yaml#L1189-L1196
 
 Notice that and others are deprecated (likely to be removed soon)